scripts/live_test/gen.py

```diff
@@ -17,7 +17,7 @@
 
 
 def main():
-    env = Environment(loader=FileSystemLoader('/home/vsts/work/1/s/scripts/live_test/'))
+    env = Environment(loader=FileSystemLoader('/mnt/vss/_work/1/s/scripts/live_test/'))
```

Comment:
Is it possible to use [predefined variables](https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml) instead of hardcoding it to `/mnt/vss/_work/1/s`?